lavalink/websocket.py

```diff
@@ -60,6 +67,17 @@ def connected(self):
                 await self._node._manager._node_connect(self._node)
                 asyncio.ensure_future(self._listen())
 
+                if not self._resuming_configured and self._resume_key is not None \
+                        and self._resume_timeout is not None:
```

Comment:
I believe that setting 0 to `_resume_timout` means that it doesn't need to resume sooo, implicit again ?